go/tendermint/apps/scheduler/scheduler.go

```diff
@@ -40,6 +41,8 @@ var (
 	rngContextMerge                = []byte("EkS-ABCI-Merge")
 	rngContextValidators           = []byte("EkS-ABCI-Validators")
 
+	RewardFactorEpochElectionAny *staking.Quantity
```

Comment:
Add documentation block. Is this something that should be part of the API or is it specific to the tendermint implementation?

Reply:
ah yeah this would be better in API land where we've been keeping the denominators

Reply:
or, more consistent, at least